docs/team/guide/run-experiments.md

```diff
@@ -0,0 +1,76 @@
+# Experiments
+
+This is a quick guide to provide some guidelines Mathesar staff should use when doing something "experimental". For example:
+
+- Adding a feature to Mathesar without going through the product process.
+- Promoting Mathesar in some previously-unknown venue.
+- Adding an integration with a 3rd-party service to Mathesar or its repo, e.g., a 1-click deploy button.
+
+We encourage experimenting with these sorts of things without the bureaucratic overhead of normal processes, but staff should try to make sure their experiment satisfies some basic guidelines.
+
+## Guidelines for an acceptable experiment
+
+These are ordered from most to least important.
+
+### Easily reversible
+
+An experiment should be easily reversible on a technical level. This means
+
+- Avoid splattering code changes related to the experiment all over the codebase. Contain them in their own directory if possible.
+- Make it obvious to other developers through comments or other means that they should not depend on the experimental code.
+- Removing any code changes associated with the experiment should be a 5-minute project leading to an easily reviewable PR.
+- Experimental GitHub workflows should be in their own file.
+
+More importantly, an experiment must be _reversible w.r.t. our perceived obligations to the public._
+
+- It should be obvious to a user that an experimental feature might disappear in the next release.
+- Experimental wrappers or integrations must not set up a long-running dependency. For example, a 1-click deploy button is fine if it doesn't require any special Docker setup. In that case, a user could continue to upgrade and use Mathesar by following our Docker instructions, even if we stop supporting a given 1-click deployment method.
+- Experimental promotions of Mathesar should not set up any obligations for the project (e.g., don't promote Mathesar at MySQL conferences by saying "we hope to support that someday").
+- Experimental installation methods should not require changes to the main Mathesar codebase.
+
+Finally, the experiment must be reversible _in your mind_. It's crucial to be honest with yourself and ask:
+
+> - If this doesn't work, will I let it go without a second thought?
+> - Am I going to be willing to admit when it isn't working?
+
+If the answer to these questions are anything but a resounding "Yes and Yes!", staff are strongly encouraged to go through normal processes rather than just taking off on their own. This brings us to the next section.
```

Comment:
```suggestion
If the answer to these questions are anything but a resounding "Yes and Yes!", staff are strongly encouraged to follow the standard product process instead of experimenting. This brings us to the next section.
```